tools/docker-compose.yml

```diff
@@ -54,18 +58,6 @@ services:
     depends_on:
       - db
 
-  # The dm admin
-  db_admin:
```

Comment:
do we not need the db_admin any more?

Reply:
I suspect no one other than myself ever used it. It is useful to see the time queries take and see the locks on the database. Want me to put it in a `developer-tools.yml` file or something?